docs/what_is_the_red_eyes_release.rst

```diff
@@ -0,0 +1,76 @@
+What is the Raiden Red Eyes Release
+************************************
+
+*And what it is not*
+
+
+.. toctree::
+  :maxdepth: 2
+
+Disclaimer!
+-----------
+**The Raiden Red Eyes release is considered safe to use on the main net. But since it is the first main net release and also acts as a bug bounty release, the amounts that can be deposited and transferred have been limited.**
+
+Introduction
+------------
+The Red Eyes release is the first version of the Raiden Network that is deployed on the Ethereum main net.
+The main goal of this release is to get the smart contracts and the core protocol battle tested on the main net.
+As part of ensuring this, a `bug bounty <https://bounty.raiden.network>`_ has been created for the release.
+Furthermore this release shows a slightly simplified version of the Raiden API, but it should suffice to get people started doing payment using the Raiden Network.
+
+The restrictions made to the Red Eyes release in order to mitigate risk in case of unexpected security issues are the following:
+
+- A user cannot create a token network. `W-ETH <https://weth.io/>`_ will be the only registered token.
+- The combined deposit of one channel is limited to 0.15 W-ETH. So 0.075 W-ETH worth of tokens per node.
+- The total combined deposit of all channels across the whole network is limited to 250 W-ETH.
+
+Below is a list of features included in the Red Eyes release along with a list of things that are scheduled for future releases and limitations of the release.
```

Comment:
The end of the sentence sounds a bit strange, I would just remove the *and limitations of the release* part.